docker/tds/docker-compose.yml

```diff
@@ -1,5 +1,20 @@
-version:  '2'
+version:  '2.1'
```

Comment:
I had to bump the docker-compose definition version to allow for health-checks.
Without a health-check, I couldn't tell the exam-service to wait until the configuration-service was fully stood up and ready to handle configuration requests.